src/v0/destinations/google_adwords_remarketing_lists/networkHandler.js

```diff
@@ -14,18 +14,24 @@ const tags = require('../../util/tags');
  * @param listId
  * @param headers
  * @param method
+ * @consentBlock
+ * ref: https://developers.google.com/google-ads/api/rest/reference/rest/v15/CustomerMatchUserListMetadata
  */
 
-const createJob = async (endpoint, customerId, listId, headers, method) => {
+const createJob = async (endpoint, customerId, listId, headers, method, consentBlock) => {
```

Comment:
did we make sure to check all the calls to this function?

Reply:
yes